packages/commons/src/config/kafkaConfig.ts

```diff
@@ -6,11 +7,15 @@ export const KafkaConfig = z
     KAFKA_CLIENT_ID: z.string(),
     KAFKA_GROUP_ID: z.string(),
     KAFKA_DISABLE_AWS_IAM_AUTH: z.literal("true").optional(),
+    KAFKA_LOG_LEVEL: z
+      .enum(["NOTHING", "ERROR", "WARN", "INFO", "DEBUG"])
+      .default("WARN"),
```

Comment:
Reusing the existing `LOG_LEVEL` variable defined in each service is better. In this case, we should change the value specified in the `.env` file for each `*-readmodel-writer` service (or for each service that consumes a kafka topic).
Otherwise, we could change only the settings for the deployed service leaving the local one to `info` level which may be it's a better approach

Reply:
I agree, reusing the existing `LOG_LEVEL` would be the better approach but here the desired is to have Kafka logs to WARN level while keeping the application ones to INFO, so the configs should be separated.

Would  you put the default as "INFO" and delegate the task to set it as "WARN" to the platform deployment?

Reply:
If we have to separate the log level between kafka and the service that uses kafka this changes it's ok!
I personally prefer to have the same level between the application and kafka but if this is the requirement then it's ok.

Reply:
@Carminepo2 is right, kafkajs logs at info level are verbose and not particularly useful.

IMO the programmatic default to `WARN` is a good approach and, only in case of need, a specific service configuration may be overridden in the deployment manifest